File: javascript/question.js
let d = 10
let a = 'nikk i'
let n = a.length
let res = ''
j = 0
for (let i = 0;i < d;i++){
    if (j>=n){
        j = 0
    }
    res += a[j]
    j+=1
}

console.log(res)

File: javascript/question.py
d = int(input("enter the diameter of the bottel :"))
strr = input("enter the string :")


n = len(strr)
res = ''
j = 0
for i in range(d):

    
    if j >= n:
        j = 0
    res += strr[j]
    j+= 1
print(res)
